File: java/parser/src/main/java/gov/lbl/parser/domain/Long_class_specifier.java
package gov.lbl.parser.domain;

public class Long_class_specifier {
    private String identifier;
	private String string_comment;
	private Composition composition;
	private Boolean is_extends;
	private Class_modification class_modification;

    public Long_class_specifier(String identifier, String string_comment, Composition composition, Boolean is_extends, Class_modification class_modification) {
        this.identifier = identifier;
        this.string_comment = string_comment;
        this.composition = composition;
        this.is_extends = is_extends;
        this.class_modification = class_modification;
    }
}

File: jsParser/domain/Long_class_specifier.js
class Long_class_specifier {
  constructor (identifier, string_comment, composition, is_extends, class_modification) {
    identifier != null && identifier != '' ? this.identifier = identifier : ''
    string_comment != null ? this.string_comment = string_comment : ''
    composition != null ? this.composition = composition : ''
    is_extends != null ? this.is_extends = is_extends : ''
    class_modification != null ? this.class_modification = class_modification : ''
  }
}
exports.Long_class_specifier = Long_class_specifier
